app/src/main/java/com/example/criminalintentapp/CrimeListFragment.kt

```diff
@@ -0,0 +1,104 @@
+package com.example.criminalintentapp
+
+import android.os.Bundle
+import android.util.Log
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.TextView
+import android.widget.Toast
+import androidx.fragment.app.Fragment
+import androidx.lifecycle.ViewModelProvider
+import androidx.recyclerview.widget.LinearLayoutManager
+import androidx.recyclerview.widget.RecyclerView
+
+class CrimeListFragment : Fragment() {
+
+    private lateinit var crimeRecyclerView: RecyclerView
+    private var adapter: CrimeAdapter? = null
+
+    private val crimeListViewModel: CrimeListViewModel by lazy {
+        ViewModelProvider(this).get(CrimeListViewModel::class.java)
+    }
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        Log.d(TAG, "Total crimes: ${crimeListViewModel.crimes.size}")
+    }
+
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+        val view = inflater.inflate(R.layout.fragment_crime_list, container, false)
+
```

Comment:
Usually I try to avoid doing UI stuff in onCreateView when it comes to fragments. The most safe way is to do it inside `onViewCreated`. Just inflating the layout should be enough here